### DISCUSSION TITLE
CondaVerificationError

### DISCUSSION THREAD

**Discussion by xchengood** [idaholab/moose#19018] [Q&A General] [answered]
Hi Moose expert or user,

When using 'conda update --all' to update my conda environment, as the download of moose-libmesh does not keep going and it is stuck there, I have to force it to break. But when running that command again, I got the CondaVerificationError.  The error message is here.
[CondaVerificationError.txt](https://github.com/idaholab/moose/files/7295458/CondaVerificationError.txt)

Could anyone help me with solving this issue? Thank you.

**Comment by GiudGiud** [comment, marked as answer]
Hello

You might have broken conda a bit by forcing it to break like this. 
Can you try cleaning the conda cache? conda clean --all
if that does not do it, you can try conda remove moose-libmesh, and if that fails too then you ll have to go in the conda pkg directory and clean manually. 

Also if you feel conda is too slow, please try out the suggestions in this thread (mamba)
https://github.com/idaholab/moose/discussions/18378

Guillaume

**Reply by xchengood** [reply]
> Hello
> 
> You might have broken conda a bit by forcing it to break like this. Can you try cleaning the conda cache? conda clean --all if that does not do it, you can try conda remove moose-libmesh, and if that fails too then you ll have to go in the conda pkg directory and clean manually.
> 
> Also if you feel conda is too slow, please try out the suggestions in this thread (mamba) #18378
> 
> Guillaume

Thank you very much. I fix issue by re-installing conda.
